proto-lens-runtime/package.yaml

```diff
@@ -0,0 +1,48 @@
+name: proto-lens-runtime
+version: '0.4.0.0'
+author: Judah Jacobson
+maintainer: proto-lens@googlegroups.com
+github: google/proto-lens/proto-lens-runtime
+copyright: Google Inc.
+license: BSD3
+description: <
```

Comment:
What does < do here vs > in some other description: > lines?

Reply:
It was a typo.  Fixed to `>`.